docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
+
+.. task:: enable_flocker_control
+   :prompt: [root@control-node]#
+
+The control service needs to accessible remotely.
+To configure firewalld to allow access to the control service REST API, and for agent connections,
 
 .. task:: open_control_firewall
 
+(On AWS, an external firewall is used instead, which will need to be configured similarity).
 For more details on configuring the firewall, see Fedora's `FirewallD documentation <https://fedoraproject.org/wiki/FirewallD>`_.
 
+To start the ZFS agent on a node, (where ``${CONTROL_NODE}`` is the address of the control node, and ``${NODE_NAME}`` is the name of the node being configured).:
```

Comment:
Is it important that this is specifically a ZFS agent? Is it always going to be?

If it is not necessary to specify ZFS then that should be removed.